.circleci/config.yml

```diff
@@ -58,7 +58,8 @@ jobs:
       - save_cache:
```

Comment:
Maybe also run,
```
ccache -s
```
after the `make`  to have some debugging information about ccache stats (e.g. to check that it's working as expected)?

Reply:
Not a bad idea.